src/components/Landing.jsx

```diff
@@ -21,19 +21,26 @@ const Landing = () => {
         className="object-cover w-full h-full absolute right-[10%] bottom-[20%] rotate-12 scale-125"
       />
 
-      <div className="absolute flex flex-col bottom-[6%] right-[3%] items-center  text-white">
-        <div className="flex flex-col items-start gap-3">
-          <h1 className="text-4xl  ml-6">EMPOWERING FUTURE INNOVATORS</h1>
+      <div className="absolute inset-0 flex flex-col justify-end items-center md:items-end md:pb-[6%] md:pr-[3%] text-white p-4">
+        <div className="flex flex-col items-center md:items-start gap-3 max-w-xl w-full">
```

Comment:
add whitespace-nowrap so it appears in one line might have to move the entire container to the left more